internal/scripts/scripts.go

```diff
@@ -82,15 +86,7 @@ func (c *ContentValue) AttrNames() []string {
 // Content methods
 // --------------------------------------------------------------------------
 
-type Check uint
-
-const (
-	CheckNone = 0
-	CheckRead = 1 << iota
-	CheckWrite
-)
-
-func (c *ContentValue) RealPath(path string, what Check) (string, error) {
+func (c *ContentValue) RealPath(path string, checkPath checkPathFunc) (string, error) {
```

Comment:
Why has that interface changed?  Before we could test Read|Write, for example, and we handed control to the function to decide how to make the choice. Now RealPath is oblivious to the operation.

Reply:
Removed from the PR.